extension/src/popup/components/manageAssets/ManageAssetRows/index.tsx

```diff
@@ -362,8 +361,10 @@ export const ManageAssetRow = ({
   image,
   domain,
 }: AssetRowData) => {
+  const { blockedDomains } = useSelector(transactionSubmissionSelector);
   const canonicalAsset = getCanonicalFromAsset(code, issuer);
-  const isScamAsset = useIsScamAsset(code, issuer);
+  const isScamAsset = !!blockedDomains.domains[domain];
```

Comment:
here was the bug fix. The assets from that search list are not ones we have trustlines for yet, which `useIsScamAsset` is using. So let's just use the domain here